lib/schema-builder/factories/enum-definition.factory.ts

```diff
@@ -1,7 +1,11 @@
+import { upperCase } from 'lodash';
 import { Injectable } from '@nestjs/common';
 import { GraphQLEnumType } from 'graphql';
 import { EnumMetadata } from '../metadata';
 
+export const mapToUppercase = (value: string): string =>
+  upperCase(value).replace(' ', '_');
```

Comment:
is this expected to replace only the first blank space?

Reply:
oh, true it should convert all blank spaces. I will fix it. Good catch

Reply:
fixed it